olog.py

```diff
@@ -61,8 +82,10 @@ async def aput_logbook(self, logbook):
             res = await api.put(f'logbooks/{logbook["name"]}',  json=logbook)
             logbook_from_server = res.json()
             logbook_from_server.pop('owner')
-            logbook.pop('owner')
-        if (not res.raise_for_status()) and (logbook != logbook_from_server):
+            logbook_cp = logbook.copy()
+            logbook_cp.pop('owner')
+        if (not res.raise_for_status()) and \
```

Comment:
This reads a little strangely because `res.raise_for_status()` either raises an exception or doesn't. Its return value's truthiness isn't really what we are interested in. Would it be OK to do:

```py
res.raise_for_status()
# The server returned OK. Its response contains a copy of what it
# inserted into its database. We can compare it with our submission for
# extra verification that everything worked correctly.
if logbook_cp != logbook_from_server:
    raise ...
```